go/pkg/amqp/url_test.go

```diff
@@ -32,13 +32,6 @@ func ExampleParseURL() {
 		"amqps://host",
 		"/path",
 		"",
-		":1234",
```

Comment:
Removing this invalid case and the commented out ones, as per the comment on [this PR](https://github.com/apache/qpid-proton/pull/231#discussion_r399227856)